models/Property/Dao.php

```diff
@@ -63,5 +63,6 @@ public function save(): void
         ];
 
         Helper::upsert($this->db, 'properties', $saveData, $this->getPrimaryKey('properties'));
+        \Pimcore\Cache::remove($this->model->getCtype() . '_properties_' . $this->model->getCid());
```

Comment:
LGTM but I think it shouldn't be in the Dao though, more like 
public function save(): void {
https://github.com/pimcore/pimcore/blob/bf89aa0d0bf0fdd0799eb1c696fdf970c3432cf3/models/DataObject/QuantityValue/Unit.php#L130-L132
}
in https://github.com/pimcore/pimcore/blob/11.2/models/Property.php?

Reply:
done